src/components/UI/Home_HeroFive.jsx

```diff
@@ -0,0 +1,42 @@
+import { motion } from "framer-motion";
+import { monsterList } from "../../vanillaJsFiles/monsters.js";
+import "./Home_HeroFive.scss";
+
+const HeroFive = (props) => {
+  return (
+    <section id="heroFive">
+      <div id="monster-container">
+        {monsterList.map((monster, index) => (
+          <motion.div
+            key={index}
+            initial={{ opacity: 0, x: -200, rotate: -90, scale: 0.25 }}
+            whileHover={{ rotate: 5, scale: 1.5 }}
+            whileTap={{ scale: 0.9 }}
+            whileInView={{ opacity: 1, x: 0, rotate: 0, scale: 1.1 }}
+            viewport={{ once: true }}
+            transition={{ duration: 0.5, delay: index * 0.2 }}
+            className="card-big"
+            style={{
+              background: `url(${monster.imgUrl}) center/cover`,
+              zIndex: monsterList.length - index,
+              left: `${(index / (monsterList.length - 1)) * 100}%`,
```

Comment:
what exactly is going on here with this equation?
i see its putting a left on it but whats the reasoning>?